### ISSUE TITLE
Human friendly name for Store creation via cli

### ISSUE THREAD

**Issue by indcoder** [openfga/openfga#809]
When creating a store via CLI, if a name is provided, then the CLI should create a a unique human friendly name automatically.

**Comment by miparnisari** [comment]
Just to clarify @indcoder, you mean if a name is _not_ provided, right?

**Comment by indcoder** [comment]
Yup, this issue cropped up in yesterdays OpenFGA community call. During the
demo noticed that if no name was provided to the CLI create  store command,
it created an alpha numeric id.

The change I propose is identical to the behaviour of Docker CLI where if
you dont provide  a name to the container while creating it, it  will
assign a unique human readable friendly name.

**Comment by miparnisari** [comment]
Ah! So this suggestion only applies to the CLI. Gotcha. I was confused because the Create Store API requires the `name` :)

**Comment by indcoder** [comment]
Exactly! CLI is mentioned in the title of the issue.
